gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -377,6 +377,21 @@
 		<type>\$type.type</type>
 		<nports>\$nchan</nports>
 	</$sourk>
+#if $sourk == 'source'
+	<source>
+		<name>rx_metad</name>
```

Comment:
Port names do not have to be 8 characters, that was MS-DOS you're thinking of. `rx_metad` is just going to confuse everyone. Let's throw in the last 3 characters (and even in MS-DOS, you got 3 more characters on the filename extension).